sdk/quantum/Azure.Quantum.Jobs/README.md

```diff
@@ -34,27 +21,43 @@ Include a section after the install command that details any requirements that m
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+In case the user already has an access token (using MSAL manually or even an external library), we will add an overload that accepts the access token string.
+
+`TokenCredential` is the default Authentication mechanism used by Azure SDKs.
 
 ## Key concepts
 
-The *Key concepts* section should describe the functionality of the main classes. Point out the most important and useful classes in the package (with links to their reference pages) and explain how those classes work together. Feel free to use bulleted lists, tables, code blocks, or even diagrams for clarity.
+'QuantumJobClient' is the root class to be used to authenticate and create, enumerate, and cancel jobs.
 
-## Examples
+'JobDetails' contains all the properties of a job.
 
-Include code snippets and short descriptions for each task you listed in the [Introduction](#introduction) (the bulleted list). Briefly explain each operation, but include enough clarity to explain complex or otherwise tricky operations.
+'ProviderStatus' contains status information for a provider.
 
-If possible, use the same example snippets that your in-code documentation uses. For example, use the snippets in your `examples.py` that Sphinx ingests via its [literalinclude](https://www.sphinx-doc.org/en/1.5/markup/code.html?highlight=code%20examples#includes) directive. The `examples.py` file containing the snippets should reside alongside your package's code, and should be tested in an automated fashion.
+'QuantumJobQuota' contains quota properties.
 
-Each example in the *Examples* section starts with an H3 that describes the example. At the top of this section, just under the *Examples* H2, add a bulleted list linking to each example H3. Each example should deep-link to the types and/or members used in the example.
+## Examples
+
+* [Get Container SAS URI](#get-container-sas-uri)
+* [Upload Input Data](#upload-input-data)
+* [Create The Job](#create-the-job)
+* [Get Job](#get-job)
+* [Get Jobs](#get-jobs)
 
-### Get the thing
+### Create the client
 
-The `get_thing` method retrieves a Thing from the service. The `id` parameter is the unique ID of the Thing, not its "name" property.
+Create an instance of the QuantumJobClient by passing in these parameters:
+- [Subscription][subscriptions] - looks like XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX and can be found in your list of subscriptions on azure
+- [Resource Group][resource-groups] - a container that holds related resources for an Azure solution 
+- [Workspace][workspaces] - a collection of assets associated with running quantum or optimization applications
+- [Location][location] - choose the best data center by geographical region 
```

Comment:
Is this scoped during preview?  Might be helpful to list the currently valid locations.